drivers/hmis_external_apis/public/schemas/referral.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$id": "open-path-schemas/referral.json",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "additionalProperties": false,
+  "title": "Referral",
+  "description": "Notify HMIS that a client and household members are eligible for a posting",
+  "type": "object",
+  "required": ["referral_id", "referral_date", "service_coordinator", "posting_id", "program_id"],
```

Comment:
unit_type_id is required as well

Reply:
Thanks @gigxz! The unit_type_id is a required field now. This made me realize I forgot to add the unit_type_id to the postings table, so I'm including a migration for it here 🤦

Reply:
Whoops yeah it wasn't in the original [db schema spec](https://docs.google.com/document/d/1qJkYoMh74fYjdsJ9p5OPT5eQDifJDNows1U8-iUTmAs/edit#heading=h.4cpn117hcidf) because we thought we would always have a Referral Request ID so we could link it that way. Now that we _don't_ alway have a referral request, the unit type needs to go on the referral or the posting as well. I think putting it on the posting is fine. Thanks!